catalog/imaging/imaging-plugin-nitf/src/test/groovy/org/codice/alliance/plugin/nitf/NitfPostIngestPluginSpec.groovy

```diff
@@ -17,11 +17,11 @@ import spock.lang.Specification
 import spock.lang.Unroll
 
 @Unroll
-class NitfPreStoragePluginSpec extends Specification {
+class NitfPostIngestPluginSpec extends Specification {
 
     def "Building derived image filename from \"#ftitle\"" (String ftitle, String expectedFname) {
         setup:
-            NitfPreStoragePlugin plugin = new NitfPreStoragePlugin();
+        NitfPostIngestPlugin plugin = new NitfPostIngestPlugin()
```

Comment:
Formatting seems inconsistent in this file. Also, maybe you could remove the semicolons on the other lines too:
```groovy
package org.codice.alliance.plugin.nitf

import spock.lang.Specification
import spock.lang.Unroll

@Unroll
class NitfPostIngestPluginSpec extends Specification {

    def "Building the derived image filename from file title \"#ftitle\""(String ftitle, String expectedFname) {
        setup:
        NitfPostIngestPlugin plugin = new NitfPostIngestPlugin()
        def qualifier = "original"

        when: "building a derived image filename"
        def derivedFname = plugin.buildDerivedImageTitle(ftitle, qualifier, "jpg")

        then: "the derived filenames should not include invalid characters"
        derivedFname == expectedFname

        where:
        ftitle                                                     || expectedFname
        null                                                       || "original.jpg"
        ""                                                         || "original.jpg"
        "_"                                                        || "original.jpg"
        "@#\$%^&*()+-={}|[]<>?:"                                   || "original.jpg"
        "Too Legit To Quit"                                        || "original-toolegittoquit.jpg"
        "A bunch of _invalid_ characters! @#\$%^&*()+-={}|[]<>?:;" || "original-abunchof_invalid_characters.jpg"
    }
}
```

Reply:
Will do